src/lib/server.ts

```diff
@@ -358,8 +358,10 @@ export class ClientsController {
     this.cleanupLoggingEventForwarder();
   }
 }
+
 interface ZwavejsServerOptions {
   port: number;
+  host?: string;
```

Comment:
this would not be optional anymore if we use the approach in the previous comment. Both port and host would default to a value so something would always be passed in

Reply:
I think `host` still needs to be optional.
I (and others) may use the class directly.

`Server.ZwavejsServer(_Driver, { .... } })`

The defaults are only applied to `bin/server.ts` aren't they?

Reply:
yes that's correct. As long as the listen command works as expected when `port` and `host` are undefined then we can make that optional here too.

Reply:
instantiating the class directly, `port` is not optional, `host` is.

using `/bin/server.ts` allows optional port and host - but it sets defaults anyway.
which satisfies the port on  the ZwavejsServer class - but also supplies a default host anyway of 0.0.0.0 - all interfaces.

If I am not mistaken, we can keep `host` as an optional parameter?

Reply:
I'm suggesting we make both parameters optional in both `/bin/server` and `/server` so that regardless of how the server is run (cmdline vs initializing the class directly) both parameters are optional

Reply:
Should be done 👍